media/src/objects/Curve.svelte

```diff
@@ -140,7 +137,8 @@
     scene.add(circleTube);
     circleTube.visible = false;
 
-    const updateCurve = function() {
+    const updateCurve = function () {
+        animation = false;
```

Comment:
These changes are undoing my animation fixes here: https://github.com/ccnmtl/mathplayground/pull/276 I don't think it was intentional - looks like a rebase problem.

Reply:
I'll try rebasing agian